docker-compose.yml

```
@@ -42,4 +122,16 @@
+      db:
+        condition: service_healthy
+      redis:
+        condition: service_healthy
+    environment:
+      - POSTGRES_HOST=db
+      - REDIS_HOST=redis
+
+volumes:
+  postgres_data:
+  redis_data:
+  minio_data:
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)